test_dependencies/Cargo.toml

```diff
@@ -20,7 +20,7 @@ tempfile = "3"
 page_size = "0.6"
 # Avoid pulling in all of tokio's dependencies.
 # However, without `net` and `signal`, tokio uses fewer relevant system APIs.
-tokio = { version = "1.24", features = ["macros", "rt-multi-thread", "time", "net", "fs", "sync", "signal"] }
+tokio = { version = "1.24", features = ["macros", "rt-multi-thread", "time", "net", "fs", "sync", "signal", "io-util"] }
```

Comment:
Which part of the test needs `io-util`?

Reply:
The tests in ``file-io.rs`` that use ``file.read`` or ``file.write`` need that.

Reply:
The [docs](https://docs.rs/tokio/1.39.3/tokio/fs/struct.File.html) say that needs the `fs` feature but don't mention `io-utils`. Strange.

Reply:
Ah, it's from the [`*Ext` traits](https://docs.rs/tokio/1.39.3/tokio/io/trait.AsyncReadExt.html).